app/pb_kits/playbook/pb_progress_pills/progress_pills.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbProgressPills
+    class ProgressPills
+      include Playbook::Props
+
+      partial "pb_progress_pills/progress_pills"
+
+      prop :active, default: 0
+      prop :text
+      prop :dark, type: Playbook::Props::Boolean,
+                 default: false
+      prop :steps, default: 3
+      prop :status
```

Comment:
Should `steps` and `number` be Number instead of String?

```
prop :active, type: Playbook::Props::Number, default: 0
prop :steps, type: Playbook::Props::Number, default: 3
```